admin_guide/managing_pods.adoc

```diff
@@ -382,15 +388,21 @@ To configure pod access limits:
 }
 ----
 +
-When the example above is added in a project, it allows traffic to `1.2.3.0/24`,
-but denies access to all other external IP addresses. (Traffic to other pods is
-not affected because the policy only applies to _external_ traffic.)
+When the example above is added in a project, it allows traffic to IP range
+`1.2.3.0/24` and domain name `www.foo.com`, but denies access to all other
+external IP addresses. (Traffic to other pods is not affected because the
+policy only applies to _external_ traffic.)
 +
 The rules in an `EgressNetworkPolicy` are checked in order, and the
-first one that matches takes effect. If the two rules in the above example
-were swapped, then traffic would not be allowed to `1.2.3.0/24` because the
-`0.0.0.0/0` rule would be checked first, and it would match and deny all
-traffic.
+first one that matches takes effect. If the three rules in the above example
+were reversed, then traffic would not be allowed to `1.2.3.0/24` and
+`www.foo.com` because the `0.0.0.0/0` rule would be checked first,
+and it would match and deny all traffic.
++
+Domain name updates are reflected within 30 mins. In the above example, lets
```

Comment:
`10.11.12.13`, but later it was changed to `20.21.22.23`. Then, {product-title} will take up to 30 minutes to adapt to these
DNS updates.